### PR TITLE
[Sprint 49]removed avro and velocity runtime dependencies

### PR DESCRIPTION
**Pull request by dturanski**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1019,7 +1019,7 @@ project('spring-xd-messagebus-kafka') {
 	dependencies {
 		compile project(':spring-xd-messagebus-spi')
 		compile("org.springframework.integration:spring-integration-kafka:$springIntegrationKafkaVersion") {
-			exclude group: 'org.apache.velocity', module: 'velocity'
+			exclude group: 'org.apache.avro', module: ':avro-compiler'
```
Comment (artembilan) [inline build.gradle]: colon typo is still here.
Have I missed anything ?

File: build.gradle [hunk 2]
```diff
@@ -1159,7 +1164,9 @@ project('spring-xd-shell') {
 		testCompile "org.apache.ftpserver:ftpserver-core:$ftpServerVersion"
 		testCompile "org.springframework.integration:spring-integration-sftp"
 		testCompile "org.apache.sshd:sshd-core:$apacheSshdVersion"
-		testCompile("org.springframework.integration:spring-integration-kafka:$springIntegrationKafkaVersion")
+		testCompile ("org.springframework.integration:spring-integration-kafka:$springIntegrationKafkaVersion") {
+			exclude group: 'org.apache.avro', module: ':avro-compiler'
```
Comment (artembilan) [inline build.gradle]: DITO

**Comment by mbogoevici** [comment]
Will fix during merge,

**Comment by dturanski** [comment]
hmmm looks like an older version for some reason. My bad

**Comment by dturanski** [comment]
Actually, the file is the latest. Those are from projects that did not
affect my testing. Sorry about that

**Comment by mbogoevici** [comment]
LGTM

**Pull request closed without merging**